dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -28,15 +28,14 @@ class CircuitPlaygroundDiscountApplication < ApplicationRecord
 
   enum unit_6_intention: {
```

Comment:
I am not 100% confident in what I've done here, and how it might affect historical submissions. Could use a closer look.